### ISSUE TITLE
Set GL interpolation method for textures

### ISSUE THREAD

**Issue by PetiteViking** [inviwo/inviwo#771]
I am currently trying to render a volume such that every voxel is shown as a brick, colored
according to the cluster it belongs to (i.e. use `GL_NEAREST` as interpolation method).

There is away to do this with something like

```c++
{
    auto interp = interpolationScheme_.get();
    auto volumeGL = volume_->getRepresentation<VolumeGL>();
    TextureUnit unit;
    volumeGL->bindTexture(unit.getEnum());
    glTexParameteri(GL_TEXTURE_3D, GL_TEXTURE_MIN_FILTER, interp);
    glTexParameteri(GL_TEXTURE_3D, GL_TEXTURE_MAG_FILTER, interp);
    volumeGL->unbindTexture();
}
```

but I think it might be nice to be able to set it as a parameter of the volume (or any texture, for
that matter) like so

```c++
auto interp = interpolationScheme_.get();
auto volumeGL = volume_->getRepresentation<VolumeGL>();
volumeGL->setInterpolationMethod(interp);
```

**Comment by martinfalk** [comment]
Definitely, but this information must be potentially stored on a per-Layer basis. Consider setting the interpolation of a GL representation and the next processor requesting a RAM representation. Then the interpolation information is lost.

See implementation of the swizzle mask!

**Comment by JochenJankowai** [comment]
Fixed in #833
